util/container/Dockerfile

```
@@ -129,6 +129,21 @@ RUN git clone https://github.com/kuleuven-micas/snitch_cluster /src && \
     make -C target/snitch_cluster bin/snitch_cluster.vlt \
     CFG_OVERRIDE=cfg/snax-gemm.hjson -j $(nproc)
 
+FROM base as snax-alu
+
+RUN git clone https://github.com/kuleuven-micas/snitch_cluster /src && \
```

Could you rename the other ones too?
```suggestion
RUN git clone https://github.com/kuleuven-micas/snax_cluster /src && \
```